ideascube/mediacenter/tests/test_clean.py

```diff
@@ -0,0 +1,141 @@
+    original_path = document.original.path
+    preview_path = document.preview.path
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    call_command('clean', 'leftover-files')
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+
+
+def test_clean_leftover_should_not_remove_files_associated_with_documents_without_preview(settings):
+    document = DocumentFactory(original__from_path=os.path.join(DATA_PATH, 'a-video.mp4'),
+                               preview=None)
+    original_path = document.original.path
+    assert os.path.exists(original_path)
+    call_command('clean', 'leftover-files')
+    assert os.path.exists(original_path)
+
+
+def test_clean_leftover_should_correctly_remove_ophan_files(settings):
+    document = DocumentFactory(original__from_path=os.path.join(DATA_PATH, 'a-video.mp4'),
+                               preview__from_path=os.path.join(DATA_PATH, 'an-image.jpg'))
+    original_path = document.original.path
+    preview_path = document.preview.path
+    Document.objects.all().delete()
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    call_command('clean', 'leftover-files')
+    assert not os.path.exists(original_path)
+    assert not os.path.exists(preview_path)
+
+
+def test_clean_leftover_should_fail_nicely_if_cannot_remove(settings, capsys):
+    document = DocumentFactory(original__from_path=os.path.join(DATA_PATH, 'a-video.mp4'),
+                               preview__from_path=os.path.join(DATA_PATH, 'an-image.jpg'))
+    original_path = document.original.path
+    preview_path = document.preview.path
+    document.delete()
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    os.chmod(os.path.join(settings.MEDIA_ROOT, 'mediacenter/document'), stat.S_IRUSR | stat.S_IXUSR)
+    call_command('clean', 'leftover-files')
+    assert os.path.exists(original_path)
+    assert not os.path.exists(preview_path)
+    _, err = capsys.readouterr()
+    assert "ERROR while deleting {}".format(original_path) in err
+
+
+def test_clean_leftover_should_not_remove_ophan_files_if_dry_run(settings, capsys):
+    document = DocumentFactory(original__from_path=os.path.join(DATA_PATH, 'a-video.mp4'),
+                               preview__from_path=os.path.join(DATA_PATH, 'an-image.jpg'))
+    original_path = document.original.path
+    preview_path = document.preview.path
+    Document.objects.all().delete()
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    call_command('clean', 'leftover-files', dry_run=True)
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    out, _ = capsys.readouterr()
+    assert original_path in out
+    assert preview_path in out
+
+
+def test_clean_leftover_should_correctly_clean_after_document_update(settings):
+    from django.core.files import File
```

Comment:
I recall last time we discussed this, we agreed that local (as in, from the code being tested, i.e ideascube in our case) imports should be done in the test functions, for lots of reasons we both agreed on.

But this is an external import, so maybe import it at the top?